atlite/wind.py

```diff
@@ -150,19 +150,20 @@ def download_turbineconf(turbine, store_locally=True):
         s = s.replace('oedb:','')
         
         # 'turbine' is just a str(-inged) id
-        m = re.match("(^\d+$)", s)
-        if m:
-            turbine['id'] = turbine.get('id', int(m[1]))
-            s = "" # Consume the string after success
+        if s.isdigit() and parsed is False:
+            turbine.setdefault('id', int(s))
             parsed = True
 
         # 'turbine' is a name or combi of manufacturer + name
-        # Matches e.g. "TurbineName", "Manu1/Manu2[_ ]Turbine/with/number"
-        m = re.match("(?P<manufacturer>[-a-zA-Z0-9\/]*?)?(?:[\s_]*)?(?P<name>[-a-zA-Z0-9\/]+)$", s)
-        if m:
-            turbine['name'] = turbine.get('name', m.group('name'))
-            turbine['manufacturer'] = turbine.get('manufacturer', m.group('manufacturer'))
-            s = "" # Consume the string after success
+        # Matches e.g. "TurbineName", "Manu1/Manu2_Turbine/number", "Man. Turb." 
+        # Split on white-spaces and underscore.
+        m = re.split("[\s|_]+",s)
```

Comment:
Last nit-pick: Is it intentional to split also on `|`, then you should also reflect this in the comment.
The `maxsplit=1`, i proposed, would additionally make sure you are not falling through the `if .. elif` below.

Reply:
Unintentional, I guess coded by habit.
Both suggestions are now included.